### PR TITLE
Apple Silicon - Update changelog

### PR DESCRIPTION
**Pull request by Jamminroot**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* ~[ ] Are tests passing locally?~
* [x] Are the files formatted correctly?
* ~[ ] Did you add unit tests?~
* ~[ ] Did you check UI tests on the sample app? They are not executed on CI.~
* ~[ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?~

## Description

This PR adds an entry regarding Apple Silicon support to changelog.

## Related PRs or issues

[AB#83370](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/83370)

### CODE REVIEW

**Review by russelarms — APPROVED** [review]

**Review by AnastasiaKubova — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### App Center
 
+* **[Feauture]** App Center SDK now supports Apple Silicon.
```
Comment (MatkovIvan) [inline CHANGELOG.md:7]: Please add a note that Simulators for arm64 is NOT available via CocoaPods. They should use other methods intead (if needed)

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 ### App Center
 
+* **[Feauture]** Add support for Apple Silicon (arm64 simulators are not supported via CocoaPods).
```
Comment (iamclement) [inline CHANGELOG.md:7]: Fix small typo :)
```suggestion
* **[Feature]** Add support for Apple Silicon (arm64 simulators are not supported via CocoaPods).
```

Comment (iamclement) [inline CHANGELOG.md:7]: The support is just for macOS modules right? We should specify this in each supported modules only then. Or mention  ''Add support....to existing macOS modules".

Comment (Jamminroot) [reply CHANGELOG.md:7]: Oh my... Thanks :D

Comment (Jamminroot) [reply CHANGELOG.md]: Well to think of it, yeah, that makes sense. I'll update it

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 ### App Center
 
+* **[Feature]** Add support for Apple Silicon to existing macOS modules (arm64 simulators are not supported via CocoaPods).
```
Comment (MatkovIvan) [inline CHANGELOG.md:7]:
```suggestion
* **[Feature]** Add support for Apple Silicon to existing macOS modules (`arm64` simulators for iOS and tvOS are not supported via CocoaPods).
```

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**